04-understanding-survey-data-documentation.Rmd

```diff
@@ -188,7 +177,7 @@ cps_state_in <- getCensus(
   name = "cps/basic/nov",
   vintage = 2020,
   region = "state",
-  vars = c("HRHHID", "HRMONTH", "HRYEAR4", "PRTAGE", "PRCITSHP", "PWSSWGT"),
+  vars = c("HRHHID", "HRMONTH", "HRYEAR4", "PRTAGE", "PRCITSHP", "PWSSWGT"), 
```

Comment:
We should provide some explanation as to what the variables are that we're pulling out and why they matter.